src/bindings/python/flux/job/list.py

```diff
@@ -97,6 +97,39 @@ def job_list_id(flux_handle, jobid, attrs=[]):
     return rpc
 
 
+# get_job is the single variant of job_list_id, and returns the
+# expected data structure (dict) to describe one job (or None)
+# This is a courtesy function for the user that looks for it
+# in the flux.job namespace after submitting a job and wanting
+# information about it.
+def get_job(flux_handle, jobid):
+    payload = {"id": int(jobid), "attrs": ["all"]}
+    rpc = JobListIdRPC(flux_handle, "job-list.list-id", payload)
+    try:
+        jobinfo = rpc.get()["job"]
+
```

Comment:
Better to move everything below this line outside of the ``try`` block?

Reply:
yeah that works, I'm indifferent about the two implementations (both have pros and cons).